File: 10 - Python/Exercícios/05 - Matrizes/Exercício 05.py
#Exercício - Soma Matrizes

linha = int(input("Quantas linhas vai ter cada matriz? "))
coluna = int(input("Quantas colunas vai ter cada matriz? "))

A = [[0 for x in range(coluna)] for x in range(linha)]
B = [[0 for x in range(coluna)] for x in range(linha)]
C = [[0 for x in range(coluna)] for x in range(linha)]

print("Digite os valores da matriz A:")
for i in range(linha):
    for j in range(coluna):
        A[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("Digite os valores da matriz B:")
for i in range(linha):
    for j in range(coluna):
        B[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("Matriz soma:")

for i in range(linha):
    for j in range(coluna):
        C[i][j] = A[i][j] + B[i][j]

for i in range(linha):
    for j in range(coluna):
        print(C[i][j], end=" ")
    print("")

File: 07 - C++/Exercícios/05 - Matrizes/Exercício 05.cpp
//Exercício - Soma Matrizes

#include <iostream>

using namespace std;

int main(){

    int i, j, linha, coluna;

    cout << "Quantas linhas vai ter cada matriz? ";
    cin >> linha;
    cout << "Quantas colunas vai ter cada matriz? ";
    cin >> coluna;

    int A[linha][coluna], B[linha][coluna], C[linha][coluna];

    cout << endl << "Digite os valores da matriz A:" << endl;
    for(i = 0; i < linha; i++){
        for(j = 0; j < coluna; j++){
            cout << "Elemento [" << i  << ", " << j << "]: ";
            cin >> A[i][j];
        }
    }

    cout << endl << "Digite os valores da matriz B:" << endl;
    for(i = 0; i < linha; i++){
        for(j = 0; j < coluna; j++){
            cout << "Elemento [" << i  << ", " << j << "]: ";
            cin >> B[i][j];
        }
    }

    for(i = 0; i < linha; i++){
        for(j = 0; j < coluna; j++){
            C[i][j] = A[i][j] + B[i][j];
        }
    }

    cout << endl << "Matriz soma:";

    for(i = 0; i < linha; i++){
        cout << endl;
        for(j = 0; j < coluna; j++){
            cout << C[i][j] << " ";
        }
    }

    return 0;

}